tests/test_helper.tcl

```diff
@@ -95,9 +95,32 @@ set ::all_tests {
     unit/violations
     unit/replybufsize
     unit/cluster/announced-endpoints
+    unit/cluster/base
     unit/cluster/misc
     unit/cluster/cli
+    unit/cluster/cluster-nodes-slots
+    unit/cluster/cluster-shards
+    unit/cluster/cluster-slots
+    unit/cluster/consistency-check
+    unit/cluster/diskless-load-swapdb
+    unit/cluster/faildet
+    unit/cluster/failover
```

Comment:
```suggestion
    unit/cluster/failover
    unit/cluster/failover-loop
```

Reply:
it look like this failover-loop will cause the test to timeout

Reply:
We should figure out why it's timing out, or leave it in the old framework for now.

Reply:
my guess is that the timeout is caused by too many `set iterations 20`. i moved it back to the old framework for now.